ssn/chapters/Actuation.html

```diff
@@ -107,17 +107,17 @@ <h5><a href="#SOSAActuation">sosa:Actuation</a></h5>
             Act of carrying out an actuating <a href="#SOSAProcedure">Procedure</a>
             using an <a href="#SOSAActuator">Actuator</a> to change the value of an actuatable
             <a href="#SOSAProperty">Property</a> of a <a href="#SOSAFeatureOfInterest">FeatureOfInterest</a>.
-            <br><br>
+        </p>
+        <p>
             In this context,
```

Comment:
The items "within" this context would probably be better presented as an unordered list than a series of paragraphs
```suggestion
        <p>
            In this context &mdash;
        </p>
        <p>
```